HW1/P4/P4.txt

```diff
@@ -1 +1,20 @@
 # Your discussion here
+
+MAXIMUM ITERATIONS
+Question: What does this diameter imply about the maximum number of steps in which your
+search must be executed?
+
+in the case of 10 nodes: 9!
+in general: (n-1)!
```

Comment:
The number of iterations should be at most D (if D is the diameter). On each iteration of the graph, the frontier increases by one, and after D iterations, the entire graph is traversed.